packages/cli/src/commands/remove.ts

```diff
@@ -167,9 +169,15 @@ export default async function main(client: Client) {
         })
       );
 
-      projectDeployments
-        .slice(0, 201)
-        .map(pDeployments => deployments.push(...pDeployments));
+      // only process the first 201 projects
+      const to = Math.min(projectDeployments.length, 201);
+      for (let i = 0; i < to; i++) {
+        for (const pDepl of projectDeployments[i]) {
+          const depl = pDepl as DeploymentWithAliases;
```

Comment:
**suggestion (non-blocking):** remove type assertion

Can we make this type check without a type assertion?

Reply:
I tried that. Actually, I believe I originally used the spread operator to copy the `Deployment` into the new `DeploymentWithAliases` type, but was an unnecessary performance hit that casting doesn't have.